src/Commands/StatusCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace UKFast\HealthCheck\Commands;
+
+use Illuminate\Console\Command;
+use UKFast\HealthCheck\Facade\HealthCheck;
+
+class StatusCommand extends Command
+{
+    protected $signature = 'health-check:status';
+
+    protected $description = 'Check health status';
+
+    public function handle()
```

Comment:
So, one thing I'd like this command to do is allow us to disable certain checks from being ran by this command per run.

If we could add a `--disable=` option, that'd be great! I imagine we'd pass the `$name` property of the check and then use that to determine whether to process the check.

https://laravel.com/docs/8.x/artisan#options-with-values

e.g. 

```php
php artisan health-check:status --disable=log,http
```
or
```php
php artisan health-check:status --disable=log --disable=http
```

Reply:
Add  `--diesable=log,cache` option